Themes/default/Errors.template.php

```diff
@@ -256,4 +256,24 @@ function template_attachment_errors()
 	</div>';
 }
 
+/**
+ * This template shows a backtrace of the given error
+ */
+function template_show_backtrace()
+{
+	global $context, $settings, $modSettings;
+
+	echo '<!DOCTYPE html>
+<html', $context['right_to_left'] ? ' dir="rtl"' : '', '>
+	<head>
+		<meta charset="', $context['character_set'], '">
+		<title>Backtrace</title>
+		<link rel="stylesheet" href="', $settings['theme_url'], '/css/index', $context['theme_variant'], '.css', $modSettings['browser_cache'], '">
+	</head>
+	<body>
+	', var_dump(!empty($context['error_backtrace'])? $context['error_backtrace'] : ''),'
```

Comment:
Yes it is, and it is easier to read.

Here is the output using `<pre>', var_dump(!empty($context['error_backtrace'])? $context['error_backtrace'] : ''), '</pre>`
![screen shot 2018-04-29 at 5 57 37 pm](https://user-images.githubusercontent.com/191731/39412266-051e6448-4bd7-11e8-80ac-fd68a548c21b.png)

Here is the output using `<pre>', print_r(!empty($context['error_backtrace'])? $context['error_backtrace'] : ''), '</pre>`
![screen shot 2018-04-29 at 5 58 13 pm](https://user-images.githubusercontent.com/191731/39412268-0dac9dfa-4bd7-11e8-9d78-cf1224e82e54.png)

Also, please wrap the output in `<pre></pre>` so that whitespace is preserved properly. Without them, the output looks like this in my tests:
![screen shot 2018-04-29 at 6 04 32 pm](https://user-images.githubusercontent.com/191731/39412335-d14ec49a-4bd7-11e8-983d-96a3e44201b8.png)

Reply:
Done also i added a css class,
so when someone like to change the style he is free todo so.